scripts/test/CrystalFieldTest.py

```diff
@@ -822,91 +822,6 @@ def test_fit_single_multi_check(self):
         fit = CrystalFieldFit(Model=cf, InputWorkspace=[ws], MaxIterations=1)
         self.assertEquals(fit.check_consistency(), None)
 
-    def test_multi_ion_single_spectrum(self):
```

Comment:
Why were all these tests removed? Are they no longer needed for some reason?

Reply:
Yes, since adding two `CrystalField` objects creates a `CrystalFieldMultiSite` now, these tests are moved to `CrystalFieldMultiSiteTest` instead (since that is the class they are actually testing).

Reply:
Just checking. I thought that was probably the case :)